draft-ietf-quic-recovery.md

```diff
@@ -1147,11 +1146,17 @@ are detected lost.
 ### On Retransmission Timeout Verified
 
 QUIC decreases the congestion window to the minimum value once the
-retransmission timeout has been verified.
+retransmission timeout has been verified and removes any packets
+sent before the newly acknowledged RTO packet.
 
 ~~~
-   OnRetransmissionTimeoutVerified()
+   OnRetransmissionTimeoutVerified(packet_number)
```

Comment:
i think OnPacketAcked() needs to be changed as well too pass in largest_sent packet right?